toonz/cmake/FindSuperLU.cmake

```diff
@@ -1,6 +1,6 @@
 # preferred homebrew's directories.
-find_path(SUPERLU_INCLUDE_DIR NAMES slu_Cnames.h HINTS ${THIRDPARTY_LIBS_HINTS} PATH_SUFFIXES superlu43/4.3_1/include/superlu superlu/SuperLU_4.1/include)
-find_library(SUPERLU_LIBRARY NAMES libsuperlu.a libsuperlu_4.1.a HINTS ${THIRDPARTY_LIBS_HINTS} PATH_SUFFIXES superlu43/4.3_1/lib superlu)
+find_path(SUPERLU_INCLUDE_DIR NAMES slu_Cnames.h HINTS ${THIRDPARTY_LIBS_HINTS} PATH_SUFFIXES superlu43/4.3_1/include/superlu superlu/SuperLU_4.1/include superlu)
+find_library(SUPERLU_LIBRARY NAMES libsuperlu.a libsuperlu_4.1.a libsuperlu.so HINTS ${THIRDPARTY_LIBS_HINTS} PATH_SUFFIXES superlu43/4.3_1/lib superlu)
```

Comment:
With this set-up you end up with mixed version (system and third party libraries) if you retain `${THIRDPARTY_LIBS_HINTS}` which I have in my setup. But if you do this, then on Linux (Ubuntu 15.10 at least) it complains that `libsuperlu.a` isn't a recognized file so I had to modify this to point to the system `superlu`. Now after doint that I was getting `relocation r_x86_64_32 recompile with -fpic` error. To overcome that I had to modify these lines like so:

find_path(SUPERLU_INCLUDE_DIR NAMES slu_Cnames.h HINTS PATH_SUFFIXES superlu43/4.3_1/include/superlu superlu/SuperLU_4.1/include superlu)
find_library(SUPERLU_LIBRARY NAMES libsuperlu.so HINTS PATH_SUFFIXES superlu43/4.3_1/lib superlu)